lib/pack.js

```diff
@@ -3,12 +3,25 @@
 const BbPromise = require('bluebird');
 const program = require('child_process');
 
+function getLambdaPackageFlags(service) {
+  var frameworkList = {
+    "dotnet6": "net6.0" //can add more mappings for more support 
+  };
+  let myFramework = ' -f ' + frameworkList[service.provider.runtime];
```

Comment:
Will it work if service.provider.runtime is not provided or it is not in the list?